containers/init/ready/ready.go

```diff
@@ -252,6 +257,50 @@ func WaitForReadyPods(ctx context.Context, ltg LoadTestGetter, pl PodLister, tes
 	return podAddresses, &nodesInfo, nil
 }
 
+func passTarget(quitServer bool, clientIP string, targets []*pb.Endpoint, testType string) error {
```

Comment:
I don't see any place where `quitServer` is supplied with a value of `false`. Any reason to keep it as a parameter?

Reply:
I forgot to add an input argument here, we can probably ask it to check if there is specific ENV, such as "QUITQUITQUIT", is so we quit the update server. Do you see a need that the user might not want to quit the test update server? If not we can probably remove that parameter and quit the test update server always.

Reply:
> Do you see a need that the user might not want to quit the test update server?
Actually, I don't understand why the update server needs to have a remote termination feature at all. Can you please help me understand this?